### PR TITLE
Add function to delete item by name

### LINKED ISSUE #53: DELETE a single item from box
## User stories
As user I want to delete a single item from an existing box.

## ToDos
- [ ] Add delete route to mongo DB
- [ ] Add button to delete item

### PR DESCRIPTION
**Pull request by Frenzy265**
close #53

### CODE REVIEW

File: lib/boxes.js [hunk 1]
```diff
@@ -15,6 +15,14 @@ async function deleteBoxByTitle(title) {
   return deleteBox;
 }
 
+async function deleteItemByName(title, item) {
+  const deletedItem = await collection("boxes").updateOne(
+    { title },
+    { $pull: { items: { $in: [item] } } }
```
Comment (kschmitz123) [inline lib/boxes.js]: What happens if you write it like this?
```suggestion
    { $pull: { items: item } }
```

**Review by Sina-Sche — COMMENTED** [review]
lgtm :) 
About your issue: it is probably similar to Sven's question: 
https://github.com/neuefische/help/issues/173
So hopefully you can fix it with Kathrin's advice too :)

File: lib/boxes.js (hunk 1 above)
Comment (Frenzy265) [reply lib/boxes.js]: Same, same. Without `$in` and without the square brackets, it delete the whole object.

Comment (Frenzy265) [reply lib/boxes.js]: Nico found the problem - I have used the wrong function at server.js 🤦🏼‍♀️

**Pull request merged**